node/src/components/contract_runtime.rs

```diff
@@ -375,6 +375,9 @@ impl ContractRuntime {
                 trace!(?request, "get era validators request");
                 let engine_state = Arc::clone(&self.engine_state);
                 let metrics = Arc::clone(&self.metrics);
+
+                self.try_init_system_contract_registry_cache();
```

Comment:
A nit, but since this function is infallible, shoud it be just `init_system_contract_registry_cache()`?

Reply:
Indeed it does not fail but it also does not guarantee that the system contract registry cache will be set either. I don't have a strong preference for naming it. However I think that if the code changes or something goes wrong a reader might be more inclined to investigate that function if it has a `try_` prefix.